### PR TITLE
fix #77936 elmueble.com

### LINKED ISSUE #77936: www.elmueble.com
### Issue URL (Anti Adblock Script)
 [https://www.elmueble.com/decoracion/brico-diy/como-colgar-tele-pladur_46541](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.elmueble.com%2Fdecoracion%2Fbrico-diy%2Fcomo-colgar-tele-pladur_46541)
### Comment
Username: @Julian1984
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.31
Browser: | Chrome
Stealth mode: | disabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Other:</b><br/> Filter unblocking search ads and self-promotion<br/><br/><b>Language-specific:</b><br/> AdGuard French, <br/>AdGuard Russian, <br/>AdGuard Chinese, <br/>AdGuard German, <br/>AdGuard Spanish/Portuguese, <br/>EasyList Italy

### PR DESCRIPTION
**Pull request by wout-codes**
#77936

### CODE REVIEW

**Comment by wout-codes** [comment]
@zloyden Should I add the domain in the scriptlet? And should the fix be in annoyances filter or in Base?

File: EnglishFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -480,8 +480,8 @@ competentedigitale.ro#@#img[width="728"][height="90"]
...
+elmueble.com,comingsoon.it,abc.es,elpais.com,idahostatesman.com,definitions.net,ensonhaber.com,elnacional.com,lapatilla.com,sozcu.com.tr,heraldo.es,20minutos.es,tucson.com,gendai.ismedia.jp,hamakei.com,telva.com,gazettetimes.com,missoulian.com,kenoshanews.com,cumberlink.com,stltoday.com,auburnpub.com,trib.com,dailyprogress.com,journalstar.com,siouxcityjournal.com,pantagraph.com#$#body { overflow: visible !important; }
+elmueble.com,comingsoon.it,abc.es,elpais.com,idahostatesman.com,definitions.net,ensonhaber.com,elnacional.com,lapatilla.com,sozcu.com.tr,heraldo.es,20minutos.es,tucson.com,gendai.ismedia.jp,hamakei.com,telva.com,gazettetimes.com,missoulian.com,kenoshanews.com,cumberlink.com,stltoday.com,auburnpub.com,trib.com,dailyprogress.com,journalstar.com,siouxcityjournal.com,pantagraph.com#$#body div.fc-ab-root { display: none !important; }
```
Comment (zloyden) [inline EnglishFilter/sections/antiadblock.txt:484]: It's ok. Also add with task comment `@@||elmueble.com^$generichide` to the `content_blocker.txt`

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/antiadblock.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/antiadblock.txt:484]: Okay.

**Comment by zloyden** [comment]
> And should the fix be in annoyances filter or in Base?

AdGuard Base.

**Review by zloyden — APPROVED** [review]

**Pull request merged**